test/fixtures/config/dummy/config.yml

```diff
@@ -4,3 +4,4 @@ zones:
   - empty.com
   - one-record.com
   - two-providers.com
+implicit_record_templates_path: templates/implicit_records/
```

Comment:
nitpick and also english is not my first language, `implicit_record_templates` vs. `implicit_records_templates` ? (i.e. could it convey the message of a single record in the first case ? in which case, we might want to add that `s` ?)

Reply:
i thiiink `implicit_record_templates` is the correct version here since each template is a template for one (type of) implicit record

Reply:
Though there could be many records injected by a single template ? `injected_record` is a list (and should probably be `injected_records` or `records_to_inject` ?)

Reply:
but each template is only for one record - how many times we do it is an implementation detail, but the template itself only specifies and generates the single record, i.e. the implicit record. grammatically `implicit_record_templates` makes more sense to me